applications/nrf_desktop/src/util/Kconfig

```diff
@@ -13,6 +13,26 @@ config DESKTOP_HWID
 	  Enable nRF Desktop hardware ID utility. The hardware ID is obtained
 	  from Zephyr hwinfo driver.
 
+
+config DESKTOP_HWINFO_BLE_ADDRESS_FICR
+	bool
+	default y
+	depends on BOARD_NRF54H20DK_NRF54H20_CPUAPP
+
+config DESKTOP_HWINFO_BLE_ADDRESS_FICR_POSTFIX
+	hex "Postfix for HW ID derived from BLE address"
+	range 0x0000 0xffff
+	default 0x0b1e
+	depends on DESKTOP_HWINFO_BLE_ADDRESS_FICR
+	help
+	  Postfix appended in big-endian to the HW ID derived from the BLE address.
+
+config HWINFO_NRF
+	bool
+	default n if BOARD_NRF54H20DK_NRF54H20_CPUAPP
+	help
+	  The nRF HWINFO driver does not support nRF54H20 yet.
```

Comment:
There seem to be no need to explicitly disable it now. It is no longer used for nRF54H.

Reply:
Are you sure?

If I remove this commit I get:

"ASSERTION FAIL [ret == 8] @ WEST_TOPDIR/nrf/applications/nrf_desktop/src/util/hwid.c:16
[54218097] <err> os: r0/a1:  0x00000004  r1/a2:  0x00000010  r2/a3:  0x00000005
[54218101] <err> os: r3/a4:  0x2f011ca8 r12/ip:  0x3fffffff r14/lr:  0x0e0aa55f
[54218104] <err> os:  xpsr:  0x41000000
[54218106] <err> os: Faulting instruction address (r15/pc): 0x0e0ac6b0
[54218127] <err> os: >>> ZEPHYR FATAL ERROR 4: Kernel panic on CPU 0
[54218136] <err> os: Current thread: 0x2f012e98 (unknown)
[54307524] <err> os: Halting system"

Reply:
Is any hwinfo driver actually compiled in after you remove the commit ?

Reply:
The test passes because it uses the dummy implementation unit for the HWinfo implementation:
hwinfo_weak_impl.c

This backend implements the HW INfo driver API by returning -ENOSYS error:

Reply:
btw. Now I think there was a bit of misunderstanding here. I just meant to remove the `default n` for `HWINFO_NRF` that is introduced here as it's no longer needed.